### PR TITLE
fix chinese document of 11 apis

### PR DESCRIPTION
**Pull request by SunGaofeng**
fix chinese document of 11 apis, including:
1, paddle.fluid.layers.dice_loss,
2, paddle.fluid.layers.clip
3, paddle.fluid.layers.crop
4, paddle.fluid.layers.ctc_greedy_decoder
5, paddle.fluid.layers.pad
6, paddle.fluid.layers.pad_constant_like
7, paddle.fluid.layers.psroi_pool
8, paddle.fluid.layers.roi_perspective_transform
9, paddle.fluid.layers.unfold
10, paddle.fluid.nets.simple_img_conv_pool
11, paddle.fluid.nets.sequence_conv_pool

1, paddle.fluid.layers.dice_loss ---------------------------

5897a5714a255baf5ed1f139d25a45ad

2, paddle.fluid.layers.clip ----------------------------------
  
ac30d26362902918b4541406dd9674d7

3, paddle.fluid.layers.crop -----------------------------------
   
80aab0e92bb55845449efd72798a3b89
 
4160993f8211f16fed3527392c7a98ab

4, paddle.fluid.layers.ctc_greedy_decoder -------------------------
  
6f069e98463380dea37917f5a9961a46
  
481683a4ab3031344afadee792a7f366

5, paddle.fluid.layers.pad-----------------------------

cf0c892a5af81b12ea2bf94666cf811f

6, paddle.fluid.layers.pad_constant_like--------------------
 
16f3eb6f7e2cb94934fda96718cb20a5

c0b2e1c9d612e44e6dddc85edc088792

7, paddle.fluid.layers.psroi_pool--------------------------

ce008ac58fd8e3992bdc744295aa5b38

8, paddle.fluid.layers.roi_perspective_transform----------------------
 
613a2de07d0afa43c3d4144d74ab0da4

9, paddle.fluid.layers.unfold------------------

7caa2fd60c1045faba26fce029774a93

7f5c7e92b03d1623c6897bd44c526aa4

10, paddle.fluid.nets.simple_img_conv_pool------------------

83ec90358a9190b46ac3b8a4437af332

47cbccd5114c6af58490ac6bfdf70680

11, paddle.fluid.nets.sequence_conv_pool--------------------

c08e85da2401a953941e2e2390217c98

### CODE REVIEW

**Comment by wanghaoshuang** [comment]
第11个op的预览有格式错误

File: doc/fluid/api_cn/nets_cn/sequence_conv_pool_cn.rst [hunk 1]
```diff
@@ -5,21 +5,24 @@ sequence_conv_pool
 
 .. py:function:: paddle.fluid.nets.sequence_conv_pool(input, num_filters, filter_size, param_attr=None, act='sigmoid', pool_type='max', bias_attr=None)
 
-sequence_conv_pool由序列卷积和池化组成
+**注意：该OP的输入 ``input`` 必须是2维LoDTensor, lod_level为1，如果输入是Tensor，建议使用 :ref:`cn_api_fluid_nets_simple_img_conv_pool` 代替**
```
Comment (wanghaoshuang) [inline doc/fluid/api_cn/nets_cn/sequence_conv_pool_cn.rst]: **符号貌似和:ref:冲突了

Comment (SunGaofeng) [reply doc/fluid/api_cn/nets_cn/sequence_conv_pool_cn.rst]: yes, modified

File: doc/fluid/api_cn/layers_cn/crop_cn.rst [hunk 2]
```diff
@@ -41,29 +41,34 @@ crop
 
 
 参数:
-  - **x** (Variable): 输入张量。
-  - **shape** (Variable|list/tuple of integer) - 输出张量的形状由参数shape指定，它可以是一个变量/整数的列表/整数元组。如果是张量变量，它的秩必须与x相同。该方式适可用于每次迭代时候需要改变输出形状的情况。如果是整数列表/tupe，则其长度必须与x的秩相同
-  - **offsets** (Variable|list/tuple of integer|None) - 指定每个维度上的裁剪的偏移量。它可以是一个Variable，或者一个整数list/tupe。如果是一个tensor variable，它的rank必须与x相同，这种方法适用于每次迭代的偏移量（offset）都可能改变的情况。如果是一个整数list/tupe，则长度必须与x的rank的相同，如果shape=None，则每个维度的偏移量为0。
-  - **name** (str|None) - 该层的名称(可选)。如果设置为None，该层将被自动命名。
+  - **x** (Variable): 1维或者多维Tensor，数据类型为float32
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/crop_cn.rst]: 写多维Tensor就可以了

File: doc/fluid/api_cn/layers_cn/dice_loss_cn.rst [hunk 3]
```diff
@@ -15,13 +15,13 @@ dice_loss定义为:
                    &= \frac{union\_area−intersection\_area}{total\_area}
 
 参数:
-    - **input** (Variable) - rank>=2的预测。第一个维度是batch大小，最后一个维度是类编号。
-    - **label** （Variable）- 与输入tensor rank相同的正确的标注数据（groud truth）。第一个维度是batch大小，最后一个维度是1。
-    - **epsilon** (float) - 将会加到分子和分母上。如果输入和标签都为空，则确保dice为1。默认值:0.00001
+    - **input** (Variable) - 分类的预测概率，rank>=2的Tensor。第一个维度的大小是batch_size，最后一个维度的大小是类别数目。数据类型是float32或者float64
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/dice_loss_cn.rst]: 下面有的地方用秩，此处用的rank，统一一下吧

File: doc/fluid/api_cn/layers_cn/pad_cn.rst [hunk 4]
```diff
@@ -28,14 +28,14 @@ pad
 
 
 参数:
-    - **x** (Variable) — —输入张量变量。
-    - **paddings** (list) — 一个整数列表。按顺序填充在每个维度上填充元素。 ``padding`` 长度必须是 ``rank(x)×2``
-    - **pad_value** (float) — 用来填充的常量值。
-    - **name** (str|None) — 这个层的名称(可选)。如果设置为None，该层将被自动命名。
+    - **x** (Variable) — 1维或者多维Tensor，数据类型为float32
+    - **paddings** (list of integers) — 整数列表，指定每个维度填充值的个数。维度 ``i`` 中 ``x`` 内容前填充的值个数用 ``paddings[2i]`` 表示，维度 ``i`` 中 ``x`` 内容后填充的值个数用 ``paddings[2i+1]`` 表示。 ``paddings`` 长度必须是 ``rank(x)×2``
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/pad_cn.rst]: paddings[2i] ？

File: doc/fluid/api_cn/layers_cn/pad_cn.rst [hunk 5]
```diff
@@ -28,14 +28,14 @@ pad
...
+    - **pad_value** (float32, 可选) — 用来填充的常量值，数据类型为float。缺省值为0.
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/pad_cn.rst]: 缺省 -> 默认

File: doc/fluid/api_cn/layers_cn/pad_constant_like_cn.rst [hunk 6]
```diff
@@ -31,15 +31,33 @@ pad_constant_like
...
+                [[0, 0, 0], 
+                 [0, 0, 0]]]]
+        out.shape = [2, 3, 2, 3]
+
 参数：
-          - **x** （Variable）- 输入Tensor变量。
-          - **y** （Variable）- 输出Tensor变量。
-          - **pad_value** (float) - 用于填充的常量值。
-          - **name** （str | None） - 这一层的名称（可选）。如果设置为None，则将自动命名这一层。
+          - **x** （Variable）- 1维或多维Tensor。
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/pad_constant_like_cn.rst]: 给一下数据类型

File: doc/fluid/api_cn/layers_cn/unfold_cn.rst [hunk 7]
```diff
@@ -23,18 +22,31 @@ unfold
...
-    - **dilations** (int|list) – 卷积膨胀，应当为[dilation_h, dilation_w]，或者一个整型的dilation处理为[dilation, dilation]。默认为[1, 1]。
+    - **x**  (Variable) – 输入Tensor，形状为[N, C, H, W]，数据类型为float32或者float64
+    - **kernel_size**  (int|list of int) – 卷积核的尺寸，整数或者整型列表。如果为整型列表，应包含两个元素 ``[k_h, k_w]`` ，卷积核大小为 ``k_h * k_w`` ；如果为整数k，会被当作整型列表 ``[k, k]`` 处理
+    - **strides**  (int|list of int，可选) – 卷积步长，整数或者整型列表。如果为整型列表，应该包含两个元素 ``[stride_h, stride_w]`` 。如果为整数，则 ``stride_h = stride_w = strides`` 。缺省值为1
+    - **paddings** (int|list of int，可选) – 每个维度的扩展, 整数或者整型列表。如果为整型列表，长度应该为4或者2；长度为4 对应的padding参数是：[padding_top, padding_left，padding_bottom, padding_right]，长度为2对应的padding参数是[padding_h, padding_w]，会被当作[padding_h, padding_w, padding_h, padding_w]处理。如果为整数padding，则会被当作[padding, padding, padding, padding]处理。缺省值为0
+    - **dilations** (int|list of int，可选) – 卷积膨胀，整型列表或者整数。如果为整型列表，应该包含两个元素[dilation_h, dilation_w]。如果是整数dilation，会被当作整型列表[dilation, dilation]处理。缺省值为1
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/unfold_cn.rst]: 缺省 -> 默认， 其他地方也改一下吧

File: doc/fluid/api_cn/layers_cn/dice_loss_cn.rst (hunk 3 above)
Comment (SunGaofeng) [reply doc/fluid/api_cn/layers_cn/dice_loss_cn.rst]: Yes, modified

File: doc/fluid/api_cn/layers_cn/pad_cn.rst (hunk 4 above)
Comment (SunGaofeng) [reply doc/fluid/api_cn/layers_cn/pad_cn.rst]: change to pad[2\*i] and pad[2\*i+1]

File: doc/fluid/api_cn/layers_cn/unfold_cn.rst (hunk 7 above)
Comment (SunGaofeng) [reply doc/fluid/api_cn/layers_cn/unfold_cn.rst]: Done

File: doc/fluid/api_cn/layers_cn/pad_constant_like_cn.rst (hunk 6 above)
Comment (SunGaofeng) [reply doc/fluid/api_cn/layers_cn/pad_constant_like_cn.rst]: These is no requirement on the dtype of x, just need the shape of x.

File: doc/fluid/api_cn/layers_cn/pad_cn.rst (hunk 5 above)
Comment (SunGaofeng) [reply doc/fluid/api_cn/layers_cn/pad_cn.rst]: done

File: doc/fluid/api_cn/layers_cn/crop_cn.rst (hunk 2 above)
Comment (SunGaofeng) [reply doc/fluid/api_cn/layers_cn/crop_cn.rst]: Done

**Review by wanghaoshuang — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Pull request merged**